### PR TITLE
Requirement Updates

### PR DESCRIPTION
**Pull request by jeff-hernandez**

### CODE REVIEW

File: requirements.txt [hunk 1]
```diff
@@ -1,4 +1,4 @@
 featuretools>=0.7.0,<0.12.0
-pandas>=0.23.0,<0.24.0
+pandas>=0.25.0
 python-dateutil>=2.6.1,<2.8.1
 tsfresh>=0.11.2
```
Comment (rwedge) [inline requirements.txt]: we should require tsfresh 0.13.0 or higher since the older versions do not support pandas 0.25

File: requirements.txt [hunk 2]
```diff
@@ -1,4 +1,4 @@
 featuretools>=0.7.0,<0.12.0
```
Comment (rwedge) [inline requirements.txt]: can you check if featuretools 0.7.0 works with pandas 0.25?  try checking out the tagged commit for 0.7.0 and running the unit tests

File: requirements.txt (hunk 1 above)
Comment (jeff-hernandez) [reply requirements.txt]: updated tsfresh version

File: requirements.txt (hunk 2 above)
Comment (jeff-hernandez) [reply requirements.txt]: locally worked with featuretools 0.7.0

**Pull request merged**